### PR TITLE
Enable all severities on the Trivy daily report

### PR DESCRIPTION
**Pull request by caponetto**
## Description
Given that not only `HIGH` and `CRITICAL` issues are worked on, let's enable all types of severity for the daily report too. This way, we can compare when a new PR that fixes an issue is opened (see [this discussion](https://github.com/opendatahub-io/notebooks/pull/613/files/f9f3586aeaddc7d6e60563e35b2bed83a98098c0#diff-6ac338da83efa67a23d1f59dcf6107320bd79f4ea30edd9ac21e48036820eb43)).

## How Has This Been Tested?

Tested locally. The default is all severities if you don't provide the `--severity` option.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by jiridanek — APPROVED** [review]

**Comment by jiridanek** [comment]
I'm imagining that the way to work with this is to first create PR without the fix, add label, trigger trivy by making dummy change in the files where I intend to make the fix, wait for trivy results, then add fix, and then compare that what I meant to fix disappeared from trivy ouput.

**Comment by caponetto** [comment]
> I'm imagining that the way to work with this is to first create PR without the fix, add label, trigger trivy by making dummy change in the files where I intend to make the fix, wait for trivy results, then add fix, and then compare that what I meant to fix disappeared from trivy ouput.

You don't need to create a PR without the fix. You can compare the report from the PR that fixes the issue with the latest daily report.

**Comment by jiridanek** [comment]
> You don't need to create a PR without the fix. You can compare the report from the PR that fixes the issue with the latest daily report.

Daily report runs with `SEVERITY_OPTION="--severity CRITICAL,HIGH"`, so if I'm fixing something less severe, I won't have it there

**Comment by caponetto** [comment]
> > You don't need to create a PR without the fix. You can compare the report from the PR that fixes the issue with the latest daily report.
> 
> Daily report runs with `SEVERITY_OPTION="--severity CRITICAL,HIGH"`, so if I'm fixing something less severe, I won't have it there

This is exactly what this PR is about 😃

File: .github/workflows/build-notebooks-TEMPLATE.yaml [hunk 1]
```diff
@@ -168,12 +168,6 @@ jobs:
           IMAGE_NAME=${{ steps.resolve-image.outputs.image }}
           echo "Scanning $IMAGE_NAME"
 
-          SEVERITY_OPTION=""
-          # Report only higher vulnerabilities if not a pull request
-          if [ "${{ fromJson(inputs.github).event_name }}" != "pull_request" ]; then
-            SEVERITY_OPTION="--severity CRITICAL,HIGH"
-          fi
```
Comment (jiridanek) [inline .github/workflows/build-notebooks-TEMPLATE.yaml:175]: you're deleting this, not adding! sorry for raving madly before

Comment (caponetto) [reply .github/workflows/build-notebooks-TEMPLATE.yaml:175]: NP! Please check the description: _The default is all severities if you don't provide the --severity option._

**Review by jstourac — APPROVED** [review]

**Pull request merged**